CMakeLists.txt

```diff
@@ -12,6 +12,17 @@ set(project HYPED)
 project(${project} CXX)
 enable_testing()
 
+include(FetchContent)
+FetchContent_Declare(
+  rapidjson
+  GIT_REPOSITORY https://github.com/Tencent/rapidjson.git
+  GIT_TAG        master
+)
+FetchContent_GetProperties(rapidjson)
+if(NOT rapidjson_POPULATED)
+    FetchContent_Populate(rapidjson)
+endif()
```

Comment:
I feel like we prefer this being locally installed now? It's what we do with eigen. Most Linux distros will have a package for rapidjson and otherwise you can install it manually.

@kshxtij ?

Reply:
Yeah i think we'd have to install it manually since the brew tap doesn't seem to work on mac 🤔

Reply:
Yeah i think we'd have to install it manually since the brew tap doesn't seem to work on mac 🤔

Reply:
and building from scratch and installing keeps throwing errors, i'll investigate this and fix it after I finish some more ROS setup for the hackathon.